docs/_src/tutorials/tutorials/5.md

```diff
@@ -289,7 +289,7 @@ More info on this metric can be found in our [paper](https://arxiv.org/abs/2108.
 
 ```python
 advanced_eval_result = pipeline.eval(
-    labels=eval_labels, params={"Retriever": {"top_k": 1}}, sas_model_name_or_path="cross-encoder/stsb-roberta-large"
+    labels=eval_labels, params={"Retriever": {"top_k": 5}}, sas_model_name_or_path="cross-encoder/stsb-roberta-large"
```

Comment:
Is this change related to the Windows problem?

Reply:
Ah interesting... We should probably review the logic behind this (moving to a git hook for example?) but let's ignore it for this PR